### PR TITLE
Feat/add homepage

### PR DESCRIPTION
**Pull request by Maianki**
## Added Homepage 
This PR includes
- [ ] Left side bar
- [ ] Right side bar
- [ ] Post component
- [ ] Filters component

### Thanks for reviewing my PR

### CODE REVIEW

**Review by SK02K1 — APPROVED** [review]
- Clean Code
- Awesome UI
- Everything is looking good, you can merge it

File: src/components/Filters/Filters.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+export function Filters() {
+  return (
+    <Menu>
+      <MenuButton as={Button} p={2} rightIcon={<BiAbacus />}>
+        Filters
+      </MenuButton>
+      <MenuList>
+        <MenuItem>Filter by trending</MenuItem>
+        <MenuItem>Filter by oldest</MenuItem>
+        <MenuItem>Filter by recent</MenuItem>
```
Comment (anik31) [inline src/components/Filters/Filters.jsx:14]: appreciation - liked that you have added a menu for filters

File: src/index.js [hunk 2]
```diff
@@ -6,16 +6,21 @@ import { makeServer } from "./server";
...
+console.log(theme.config.initialColorMode);
 root.render(
   <StrictMode>
     <Provider store={store}>
-      <ChakraProvider>
+      <ChakraProvider
+        theme={myNewTheme}
+        initialColorMode={theme.config.initialColorMode}
+      >
         <BrowserRouter>
```
Comment (anik31) [inline src/index.js:24]: suggestion - browser router should be at the top

**Review by anik31 — APPROVED** [review]
Nice work, the homepage looks good. 

suggestion - you can remove the `home.css` file if not needed

File: src/index.js (hunk 2 above)
Comment (Maianki) [reply src/index.js:24]: It does not matter here as they are not related to each other.

Comment (Maianki) [reply src/index.js:24]: I think here it does not matter as they are not dependent on each other.

**Pull request merged**